interface/forms/procedure_order/new.php

```diff
@@ -214,34 +214,19 @@ function getListOptions($list_id , $fieldnames=array('option_id', 'title', 'seq'
 <head>
 <?php html_header_show(); ?>
 
-<?php $include_standard_style_js = array("datetimepicker"); ?>
-<?php require($GLOBALS['srcdir'] . '/templates/standard_header_template.php'); ?>
```

Comment:
Keep these 2 lines, which will then include all css/js stuff except for the dialog.js and textformat.js includes.